pkg/nanopb/nanopb.inc.mk

```diff
@@ -0,0 +1,22 @@
+PROTOC ?= protoc
+PROTOC_GEN_NANOPB ?= protoc-gen-nanopb
+
+PROTOBUF_FILES ?= $(wildcard *.proto)
+GENSRC   += $(PROTOBUF_FILES:%.proto=$(BINDIR)/$(MODULE)/%.pb.c)
+GENOBJC  := $(GENSRC:%.c=%.o)
+
+ifneq (, $(PROTOBUF_FILES))
+  INCLUDES += -I$(BINDIR)/$(MODULE)
+endif
+
+$(SRC): $(GENSRC)
+
+$(GENSRC): $(PROTOBUF_FILES)
+	$(Q)D=$(BINDIR)/$(MODULE) && \
+	  mkdir -p "$$D" && \
+		cd $(CURDIR) && \
```

Comment:
`make -C $(PKGDIRBASE)/nanopb/generator/proto && \`

Doesn't hurt and makes for a smoother user experience.